### PR TITLE
GDB-8023: implement "Explain plan query" shortcut action.

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
Implement "Explain plan query" action.

## Why
The WB have got implementation of this action.

## How

### CODE REVIEW

File: cypress/e2e/keyboard-shortcuts.spec.cy.ts [hunk 1]
```diff
@@ -225,15 +226,25 @@ describe('Keyboard Shortcuts', () => {
...
 
       // When press the "Create saved query" keyboard shortcut.
       KeyboardShortcutSteps.clickOnCreateQueryShortcut();
 
       // Then I expect "Create New Saved Query" to be visible.
-      YasqeSteps.getCreateSavedQueryButton().should('be.visible');
+      YasqeSteps.getSaveQueryDialog().should('be.visible');
+    });
+
+    it.only('should trigger "EXECUTE_EXPLAIN_PLAN_FOR_QUERY" action', () => {
```
Comment (svilenvelikov) [inline cypress/e2e/keyboard-shortcuts.spec.cy.ts:240]: only

File: Yasgui/packages/yasqe/src/index.ts [hunk 2]
```diff
@@ -945,10 +954,29 @@ export class Yasqe extends CodeMirror {
   /**
    * Querying
    */
-  public query(config?: Sparql.YasqeAjaxConfig) {
+  public query(config?: Sparql.YasqeAjaxConfig, explain = false) {
     if (this.config.queryingDisabled) return Promise.reject("Querying is disabled.");
+    const queryType = this.getQueryType()?.toLowerCase();
+
+    if (explain && !("select" === queryType || "construct" === queryType)) {
```
Comment (svilenvelikov) [inline Yasgui/packages/yasqe/src/index.ts:961]: You can create methods like `isSelectQuery`, `isConstructQuery` etc.

File: Yasgui/packages/yasqe/src/index.ts [hunk 3]
```diff
@@ -945,10 +954,29 @@ export class Yasqe extends CodeMirror {
   /**
    * Querying
    */
-  public query(config?: Sparql.YasqeAjaxConfig) {
+  public query(config?: Sparql.YasqeAjaxConfig, explain = false) {
```
Comment (svilenvelikov) [inline Yasgui/packages/yasqe/src/index.ts:957]: `explain` is not exactly conventional name for a boolean variable. Would you consider renaming it to `isExplainQuery` or something similar?

[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply Yasgui/packages/yasqe/src/index.ts:957]: I will remove undefined check, it is left from my first implementation.

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**